libs/core/langchain_core/runnables/fallbacks.py

```diff
@@ -343,3 +347,74 @@ async def abatch(
             raise ValueError("No error stored at end of fallbacks.")
         await asyncio.gather(*(rm.on_chain_error(first_error) for rm in run_managers))
         raise first_error
+
+    def stream(
+        self,
+        input: Input,
+        config: Optional[RunnableConfig] = None,
+        **kwargs: Optional[Any],
+    ) -> Iterator[Output]:
+        # setup callbacks
+        config = ensure_config(config)
+        callback_manager = get_callback_manager_for_config(config)
+        # start the root run
+        run_manager = callback_manager.on_chain_start(
+            dumpd(self), input, name=config.get("run_name")
+        )
+        first_error = None
+        for runnable in self.runnables:
+            try:
+                stream = runnable.stream(
+                    input,
+                    patch_config(config, callbacks=run_manager.get_child()),
+                    **kwargs,
+                )
+                chunk = next(stream)
+            except self.exceptions_to_handle as e:
+                first_error = e if first_error is None else first_error
```

Comment:
We have an `on_retry` callback. I was expecting to see it used within this Runnable, but don't see it anywhere. Do we need to add it to the various calls or do we think it doesn't make sense for this runnable or some other issue?